docs/command-reference/server-management/save.md

```diff
@@ -6,20 +6,15 @@ description: Synchronously save the dataset to disk
 
 ## Syntax
 
-    SAVE 
+    SAVE [RDB|DF]
 
 **Time complexity:** O(N) where N is the total number of keys in all databases
 
-The `SAVE` commands performs a **synchronous** save of the dataset producing a
-_point in time_ snapshot of all the data inside the Redis instance, in the form
-of an RDB file.
+The `SAVE` commands performs a save of the dataset producing a
+_point in time_ snapshot of all the data inside the Dragonfly instance, in the form
+of a collection of DFS files.
```

Comment:
collection of dfs files sounds not good

Reply:
"DFS" abbreviation must be explained. collection -> group -> set ?

Reply:
Lets call it 'set of DFS files' and make it point to the serialization page